components/dash-table/src/dash-table/dash/DataTable.js

```diff
@@ -110,17 +110,20 @@ export const defaultProps = {
 
 export const propTypes = {
     /**
-     * The row and column indices and IDs of the currently active cell.
-     * `row_id` is only returned if the data rows have an `id` key.
+     * The contents of the table.
+     * The keys of each item in data should match the column IDs.
+     * Each item can also have an 'id' key, whose value is its row ID. If there
+     * is a column with ID='id' this will display the row ID, otherwise it is
+     * just used to reference the row for selections, filtering, etc.
+     * Example:
+     * [
+     *      {'column-1': 4.5, 'column-2': 'montreal', 'column-3': 'canada'},
+     *      {'column-1': 8, 'column-2': 'boston', 'column-3': 'america'}
+     * ]
      */
-    active_cell: PropTypes.exact({
-        row: PropTypes.number,
-        column: PropTypes.number,
-        row_id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-        column_id: PropTypes.string
-    }),
+    data: PropTypes.arrayOf(PropTypes.object),
 
-    /**
+     /**
```

Comment:
NBD, but I think the convention in the rest of the comments is for this to have a single tab and for the following `*` to have a tab plus one space.